ironfish/src/network/testUtilities/helpers.ts

```diff
@@ -156,8 +150,24 @@ export function getSignalingWebRtcPeer(
   const peer = pm.getOrCreatePeer(peerIdentity)
 
   // Link the peers
-  brokeringPeer.knownPeers.set(peerIdentity, peer)
-  peer.knownPeers.set(brokeringPeerIdentity, brokeringPeer)
+  pm.peerCandidateMap.set(brokeringPeerIdentity, {
+    address: brokeringPeer.address,
+    port: brokeringPeer.port,
+    neighbors: new Set([peerIdentity]),
+    webRtcRetry: new ConnectionRetry(),
+    websocketRetry: new ConnectionRetry(),
+    peerRequestedDisconnectUntil: null,
+    localRequestedDisconnectUntil: null,
+  })
+  pm.peerCandidateMap.set(peerIdentity, {
```

Comment:
Do we want to reset everything like the `webRtcRetry` and `websocketRetry` here? What if the peer already exists?

Reply:
This is just used in a test helper and the peer should never exist at this point. Adding an `Assert` to make sure that's true